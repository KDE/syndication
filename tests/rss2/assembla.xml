<?xml version="1.0" encoding="UTF-8"?>
<!-- HTML detected as plaintext 
    bug 172845 -->
<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:trackback="http://www.assembla.com">
  <channel>
    <title>Assembla Breakout Software Development Alerts</title>
    <link>https://www.assembla.com/spaces/new_items/breakout</link>
    <description>New items and changes in Breakout Software Development space</description>
    <language>en-us</language>
    <ttl>60</ttl>
    <item>
      <title>Ticket #2871 updated: User choise is not persisted in </title>
      <description>&lt;b&gt;#2871:&lt;/b&gt; User choise is not persisted in &lt;br&gt;---------------------------+-------------------------------------------------&lt;br&gt; Reporter: sromano         |     Owner: sreenath            &lt;br&gt;   Status: Accepted        |  Priority: Normal (3)          &lt;br&gt;Milestone: Metered Billing | Component: Preconfigured Spaces&lt;br&gt;---------------------------+-------------------------------------------------&lt;br&gt;Changes (by sreenath s):&lt;br&gt;  * assigned_to:  =&gt; sreenath&lt;br&gt;  * status: New =&gt; Accepted&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;a href='https://www.assembla.com/spaces/b6nRoCPC0r2Oq0aaeP0Qfc/tickets/2871'&gt;https://www.assembla.com/spaces/b6nRoCPC0r2Oq0aaeP0Qfc/tickets/2871&lt;/a&gt;</description>
      <pubDate>Wed, 15 Oct 2008 08:44:28 -0000</pubDate>
      <id>6624723</id>
      <author>sreenath</author>
      <link>http://www.assembla.com/spaces/breakout/tickets/2871</link>
    </item>
    <item>
      <title>Scrum Report created: I'll finish with #49 and can handle another tasks.</title>
      <description>Scrum report from Igor Petrushenko&lt;br&gt;&lt;br&gt;What I did in the last day / week&lt;br&gt;Worked on ticket #49. Testing breakout and monitoring servers.&lt;br&gt;&lt;br&gt;What I will do today / this week&lt;br&gt;I'll finish with #49 and can handle another tasks.&lt;br&gt;&lt;br&gt;What are my roadblocks and what do I need?&lt;br&gt;&lt;br&gt;&lt;a href='https://www.assembla.com/spaces/user_report/ciWstCAjur3A1lab7jnrAJ?space_id=b6nRoCPC0r2Oq0aaeP0Qfc'&gt;https://www.assembla.com/spaces/user_report/ciWstCAjur3A1lab7jnrAJ?space_id=b6nRoCPC0r2Oq0aaeP0Qfc&lt;/a&gt;</description>
      <pubDate>Wed, 15 Oct 2008 05:22:20 -0000</pubDate>
      <id>6623258</id>
      <author>pro_igor</author>
      <link>https://www.assembla.com/spaces/scrum/index/b6nRoCPC0r2Oq0aaeP0Qfc?current_date=2008-10-15&amp;amp;report_id=23649</link>
    </item>
    <item>
      <title>Ticket #2867 updated: Create an upgrade page to decide about all not suported spaces for users</title>
      <description>&lt;b&gt;#2867:&lt;/b&gt; Create an upgrade page to decide about all not suported spaces for users&lt;br&gt;---------------------------+-------------------------------------------------&lt;br&gt; Reporter: sromano         |     Owner: sromano         &lt;br&gt;   Status: Test            |  Priority: High (2)        &lt;br&gt;Milestone: Metered Billing | Component: Home / Marketing&lt;br&gt;---------------------------+-------------------------------------------------&lt;br&gt;Changes (by Sergio Romano):&lt;br&gt;  * status: Accepted =&gt; Test&lt;br&gt;&lt;br&gt;Comment:&lt;br&gt;(In [b6d9163442ffeef3d48ef089a8dc1e22c8b5ed14]) Create an upgrade page to decide about all not suported spaces for users&lt;br&gt;test #2867&lt;br&gt;&lt;br&gt;Branch:master&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;a href='https://www.assembla.com/spaces/b6nRoCPC0r2Oq0aaeP0Qfc/tickets/2867'&gt;https://www.assembla.com/spaces/b6nRoCPC0r2Oq0aaeP0Qfc/tickets/2867&lt;/a&gt;</description>
      <pubDate>Tue, 14 Oct 2008 23:23:08 -0000</pubDate>
      <id>6621261</id>
      <author>Sergio Romano</author>
      <link>http://www.assembla.com/spaces/breakout/tickets/2867</link>
    </item>
  </channel>
</rss>
