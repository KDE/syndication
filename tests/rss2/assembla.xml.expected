# Feed begin ######################
title: #Assembla Breakout Software Development Alerts#
link: #https://www.assembla.com/spaces/new_items/breakout#
description: #New items and changes in Breakout Software Development space#
language: #en-us#
# Item begin ######################
id: #hash:fcfdb822d4f766388027d39e86a493ee#
title: #Ticket #2871 updated: User choise is not persisted in#
link: #http://www.assembla.com/spaces/breakout/tickets/2871#
description: #<b>#2871:</b> User choise is not persisted in <br>---------------------------+-------------------------------------------------<br> Reporter: sromano         |     Owner: sreenath            <br>   Status: Accepted        |  Priority: Normal (3)          <br>Milestone: Metered Billing | Component: Preconfigured Spaces<br>---------------------------+-------------------------------------------------<br>Changes (by sreenath s):<br>  * assigned_to:  => sreenath<br>  * status: New => Accepted<br><br><br><br><a href='https://www.assembla.com/spaces/b6nRoCPC0r2Oq0aaeP0Qfc/tickets/2871'>https://www.assembla.com/spaces/b6nRoCPC0r2Oq0aaeP0Qfc/tickets/2871</a>#
datePublished: #Wed Oct 15 10:44:28 2008#
dateUpdated: #Wed Oct 15 10:44:28 2008#
# Person begin ####################
name: #sreenath#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #hash:56eaedfd6f8576a8302059e2dc530838#
title: #Scrum Report created: I&apos;ll finish with #49 and can handle another tasks.#
link: #https://www.assembla.com/spaces/scrum/index/b6nRoCPC0r2Oq0aaeP0Qfc?current_date=2008-10-15&amp;report_id=23649#
description: #Scrum report from Igor Petrushenko<br><br>What I did in the last day / week<br>Worked on ticket #49. Testing breakout and monitoring servers.<br><br>What I will do today / this week<br>I'll finish with #49 and can handle another tasks.<br><br>What are my roadblocks and what do I need?<br><br><a href='https://www.assembla.com/spaces/user_report/ciWstCAjur3A1lab7jnrAJ?space_id=b6nRoCPC0r2Oq0aaeP0Qfc'>https://www.assembla.com/spaces/user_report/ciWstCAjur3A1lab7jnrAJ?space_id=b6nRoCPC0r2Oq0aaeP0Qfc</a>#
datePublished: #Wed Oct 15 07:22:20 2008#
dateUpdated: #Wed Oct 15 07:22:20 2008#
# Person begin ####################
name: #pro_igor#
# Person end ######################
# Item end ########################
# Item begin ######################
id: #hash:5231a17222c786104095a1707e46d24f#
title: #Ticket #2867 updated: Create an upgrade page to decide about all not suported spaces for users#
link: #http://www.assembla.com/spaces/breakout/tickets/2867#
description: #<b>#2867:</b> Create an upgrade page to decide about all not suported spaces for users<br>---------------------------+-------------------------------------------------<br> Reporter: sromano         |     Owner: sromano         <br>   Status: Test            |  Priority: High (2)        <br>Milestone: Metered Billing | Component: Home / Marketing<br>---------------------------+-------------------------------------------------<br>Changes (by Sergio Romano):<br>  * status: Accepted => Test<br><br>Comment:<br>(In [b6d9163442ffeef3d48ef089a8dc1e22c8b5ed14]) Create an upgrade page to decide about all not suported spaces for users<br>test #2867<br><br>Branch:master<br><br><br><br><a href='https://www.assembla.com/spaces/b6nRoCPC0r2Oq0aaeP0Qfc/tickets/2867'>https://www.assembla.com/spaces/b6nRoCPC0r2Oq0aaeP0Qfc/tickets/2867</a>#
datePublished: #Wed Oct 15 01:23:08 2008#
dateUpdated: #Wed Oct 15 01:23:08 2008#
# Person begin ####################
name: #Sergio Romano#
# Person end ######################
# Item end ########################
# Feed end ########################

